vms/event/tests/test_shiftSignUp.py

```diff
@@ -172,77 +157,77 @@ def test_shift_sign_up_with_outdated_shifts(self):
 
         # on jobs page
         sign_up_page.click_to_view_shifts()
-        self.assertEqual(sign_up_page.get_info_box().text,'There are currently no shifts for the job job.')
+        self.assertEqual(sign_up_page.get_info_box().text, 'There are currently no shifts for the job job.')
 
     def test_shift_sign_up_with_no_slots(self):
-
         register_event_utility()
         register_job_utility()
 
         sign_up_page = self.sign_up_page
 
         # create shift with no slot
-        shift_2 = ["2016-05-11","9:00","15:00",1,Job.objects.get(name = 'job')]
+        shift_2 = ["2050-05-11", "9:00", "15:00", 1, Job.objects.get(name='job')]
         s2 = create_shift_with_details(shift_2)
 
         # Create another volunteer
-        volunteer_2 = ['volunteer-2',"Sam","Turtle","Mario Land","Nintendo Land","Nintendo State","Nintendo Nation","2374983247","volunteer2@volunteer.com"]
+        volunteer_2 = ['volunteer-2', "Sam", "Turtle", "Mario Land", "Nintendo Land", "Nintendo State",
+                       "Nintendo Nation", "2374983247", "volunteer2@volunteer.com"]
         v2 = create_volunteer_with_details(volunteer_2)
 
         # Assign shift to the volunteer
-        vol_shift = register_volunteer_for_shift_utility(s2, v2)
+        register_volunteer_for_shift_utility(s2, v2)
```

Comment:
Capture return value

Reply:
codacy was marking it as unused variable and failing. :disappointed: